[2023-05-02T07:02:54.741+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-05-02T07:02:52.396312+00:00 [queued]>
[2023-05-02T07:02:54.761+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-05-02T07:02:52.396312+00:00 [queued]>
[2023-05-02T07:02:54.761+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-02T07:02:54.761+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-02T07:02:54.761+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-02T07:02:54.791+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-05-02 07:02:52.396312+00:00
[2023-05-02T07:02:54.797+0000] {standard_task_runner.py:55} INFO - Started process 20634 to run task
[2023-05-02T07:02:54.801+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-05-02T07:02:52.396312+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/processing_user.py', '--cfg-path', '/tmp/tmpr6i40gi1']
[2023-05-02T07:02:54.803+0000] {standard_task_runner.py:83} INFO - Job 65: Subtask create_table
[2023-05-02T07:02:54.911+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2023-05-02T07:02:52.396312+00:00 [running]> on host 9bd2cedcb1a8
[2023-05-02T07:02:55.048+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-05-02T07:02:52.396312+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-02T07:02:52.396312+00:00
[2023-05-02T07:02:55.063+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-05-02T07:02:55.068+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS USERS (
            firstname text not null,
            lastname text not null,
            country text not null,
            username text not null,
            password text not null,
            email text not null
            )
        , parameters: None
[2023-05-02T07:02:55.070+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-05-02T07:02:55.083+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230502T070252, start_date=20230502T070254, end_date=20230502T070255
[2023-05-02T07:02:55.133+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-02T07:02:55.162+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
